etc/kayobe/kolla.yml

```diff
@@ -541,7 +541,7 @@ kolla_build_customizations: "{{ kolla_build_customizations_common | combine(koll
 #kolla_enable_neutron:
 #kolla_enable_neutron_agent_ha:
 #kolla_enable_neutron_bgp_dragent:
-#kolla_enable_neutron_dvr:
+kolla_enable_neutron_dvr: yes
```

Comment:
This flag is for neutron OVS DVR.

We need to set `neutron_ovn_distributed_fip: true` in `etc/kayobe/kolla/globals.yml`. See https://docs.openstack.org/kolla-ansible/latest/reference/networking/neutron.html#ovn-ml2-ovn

Reply:
Thanks, will do.